### ISSUE TITLE
Prometheus query editor missing dashboard variables in label values suggestions

### ISSUE THREAD

**Issue by adeverteuil** [grafana/grafana#30656]
**What happened**:

I created a `myvar` variable (custom type with CSV values "a,b,c").

When I write a PromQL query, the `$myvar` variable is not in the list of suggested values.

![Screenshot 2021-01-26 15-40-05](https://user-images.githubusercontent.com/3270755/105903358-c744e100-5fed-11eb-9d67-e4f0483069eb.png)

**What you expected to happen**:

I remember this worked in a previous version of Grafana.

When I tried with the Graphite data source, the `$myvar` variable is present in the suggestions dropdown:

![Screenshot 2021-01-26 15-39-39](https://user-images.githubusercontent.com/3270755/105902930-44bc2180-5fed-11eb-91b1-9128e4b045fb.png)

**How to reproduce it (as minimally and precisely as possible)**:

1. Create a dashboard with a variable.
2. Create a panel, select a Prometheus data source.
3. Write a simple query until the label values suggestions box appears.

**Anything else we need to know?**:

**Environment**:
- Grafana version: v7.3.7
- Data source type & version: Prometheus
- OS Grafana is installed on: Linux, binary tarball
- User OS & Browser: Firefox 84 on Fedora 33
- Grafana plugins:
- Others:

**Comment by ivanahuckova** [comment]
@adeverteuil I don't remember this functionality for Prometheus, but maybe I am wrong. I have tried this in Grafana 7.2. but it is not there either. Do you remember in which version it was? 
![image](https://user-images.githubusercontent.com/30407135/105998433-ece4ef80-60ac-11eb-8878-4721a7bb7399.png)

**Comment by adeverteuil** [comment]
I had to go back to v6.0.1 where the suggestions list only showed up as you started typing the string inside the quotation marks. If you started with a `$`, then you get a list of dashboard variables.

![Screenshot 2021-01-27 15-33-54](https://user-images.githubusercontent.com/3270755/106050585-579a2900-60b5-11eb-9c62-89146b76e711.png)

I tried v6.{1,2,3,4}.1. The feature was removed/broken between v6.2.1 and v6.3.1
